remote/gitlab/gitlab.go

```diff
@@ -198,10 +198,16 @@ func (g *Gitlab) Perm(u *model.User, owner, name string) (*model.Perm, error) {
 	if err != nil {
 		return nil, err
 	}
+	
+	currentUser, err := client.CurrentUser()
+	if err != nil {
```

Comment:
what about using `u.Login` (the user login passed into the function) instead of making an axtra API call?